nixos/modules/installer/cd-dvd/installation-cd-base.nix

```diff
@@ -30,5 +30,16 @@ with lib;
   # Add Memtest86+ to the CD.
   boot.loader.grub.memtest86.enable = true;
 
+  boot.postBootCommands = ''
+    for o in $(</proc/cmdline); do
+      case "$o" in
+        live.nixos.passwd=*)
```

Comment:
Our cloud-init integration should allow to set a password: https://cloudinit.readthedocs.io/en/latest/topics/examples.html
We have cloud-init enabled in various cloud image profiles.
What is your workflow for changing kernel arguments on headless machines? I think this deserves some documentation. Maybe it should be even generalized to `nixos.password` and be put into its own module.

Reply:
This is specifically for the normal installation image, to aid in debugging if, for example, your GPU drivers don't work out of the box (without having to recompile the installation medium with a pre-configured password). But the machine isn't (necessarily) headless. In the end I was able to install from Plasma using a custom kernel command line option `pci=noats` (which I can set because the GPU only stops working after AMD drivers are loaded). I wouldn't have been able to figure out what the problem was without an interactive shell, and using this I could get an interactive shell over SSH.

Reply:
Ok. I still would insist on documentation in the manual here that this flag insists and what the workflow is to change it given an nixos image.

Reply:
Is this okay? I didn't find any other places in the manual to document it.